mobile/lib/pages/editing/crop.page.dart

```diff
@@ -0,0 +1,185 @@
+                          },
+                        ),
+                        IconButton(
+                          icon: const Icon(Icons.rotate_right,
+                              color: Colors.white),
+                          onPressed: () {
+                            _cropController.rotateRight();
+                          },
+                        ),
+                      ],
+                    ),
+                  ),
+                  Row(
+                    mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+                    children: <Widget>[
+                      Column(
+                        mainAxisSize: MainAxisSize.min,
+                        children: [
+                          IconButton(
+                              icon: Icon(Icons.crop_free_rounded,
+                                  color: _selectedAspectRatio == null
+                                      ? Colors.blue
+                                      : Colors.white),
+                              onPressed: () => setState(() {
+                                    _cropController.aspectRatio = null;
+                                    _selectedAspectRatio = null;
+                                  })),
+                          const Text('Free',
+                              style: TextStyle(color: Colors.white)),
+                        ],
+                      ),
+                      Column(
+                        mainAxisSize: MainAxisSize.min,
+                        children: [
+                          IconButton(
+                              icon: Icon(Icons.crop_square_rounded,
+                                  color: _selectedAspectRatio == 1
+                                      ? Colors.blue
+                                      : Colors.white),
+                              onPressed: () => setState(() {
+                                    _cropController.aspectRatio = 1;
+                                    _selectedAspectRatio = 1;
+                                  })),
+                          const Text('1:1',
+                              style: TextStyle(color: Colors.white)),
+                        ],
+                      ),
+                      Column(
+                        mainAxisSize: MainAxisSize.min,
+                        children: [
+                          IconButton(
+                              icon: Icon(Icons.crop_16_9_rounded,
+                                  color: _selectedAspectRatio == 16 / 9
+                                      ? Colors.blue
+                                      : Colors.white),
+                              onPressed: () => setState(() {
+                                    _cropController.aspectRatio = 16 / 9;
+                                    _selectedAspectRatio = 16 / 9;
+                                  })),
+                          const Text('16:9',
+                              style: TextStyle(color: Colors.white)),
```

Comment:
Using explicit `TextStyle(color: ...)` may make the text invisible in bright mode / dark mode. Usually better to use `Theme.of(context).textTheme` and pick a text style from there. You will use the same font, and also get the correct brightness-enabled color.

Reply:
Done, `Text(label, style: Theme.of(context).textTheme.bodyMedium)`